Tickr/Tickr/SearchTableViewController.swift

```diff
@@ -0,0 +1,141 @@
+//
+//  SearchTableViewController.swift
+//  Tickr
+//
+//  Created by Stephen Payne on 4/10/16.
+//  Copyright © 2016 Stephen Payne. All rights reserved.
+//
+
+import UIKit
+
+class SearchTableViewController: UIViewController {
+    //Class Variables
+    var searchResults: [StockSearchResult] = []
+    
+    //Outlets
+    @IBOutlet weak var tableView: UITableView!
+    @IBOutlet weak var searchBar: UISearchBar!
+    
+    //MARK: - View Lifecycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        searchBar.delegate = self
+    }
+    
+    override func viewDidAppear(animated: Bool) {
+        super.viewDidAppear(animated)
+        
+        searchBar.becomeFirstResponder()
+    }
+
+    /*
+        Will take a search term and query a separate API for possible stocks
+        that match that search term.
+     
+        - parameter searchText: The user input used for finding stocks
+    */
+    func searchYahooFinanceWithString(searchText: String) {
+        
+        //Search for stocks using the user-entered text
+        StockManager.fetchStocksFromSearchTerm(term: searchText) { (stockInfoArray) -> () in
+            dispatch_async(dispatch_get_main_queue(), {
+                //Update the tableView with the search results
+                self.searchResults = stockInfoArray
+                self.tableView.reloadData()
+            })
+        }
+    }
+}
+
+//MARK: - TableView Delegate and Datasource
+extension SearchTableViewController: UITableViewDelegate, UITableViewDataSource {
+    
+    func tableView(tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return searchResults.count
+    }
+    
+    func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCellWithIdentifier("searchResultCell") as! SearchResultCell
+        cell.parentVC = self
+        cell.configureCellWithSearchResult(searchResults[indexPath.row])
+        
+        return cell
+    }
+    
+    func tableView(tableView: UITableView, didSelectRowAtIndexPath indexPath: NSIndexPath) {
+        let cell = tableView.cellForRowAtIndexPath(indexPath) as! SearchResultCell
+        cell.addButtonWasPressed(nil)
```

Comment:
`addButtonWasPressed` is just a bad method name, it makes you think you are getting and setting a boolean, but you are not, which is weird